pages/background.md

```diff
@@ -1,25 +1,30 @@
 
-YOU SHOULD BE LIBERAL AND INCLUDE ANY AND ALL LINKS WHEN RELEVANT
 
 ---
 
+![image1]({{ site.baseurl }}/assets/images/background picture.jpg)
+
 ## Historical Background
 
-This is where your historiography goes. You can discuss the historiography of specific times and places, but you can also discuss any theoretical literature you think is relevant.
+Although the history of piracy is often considered small than that of the Trans-Atlantic triangular trade, the social and cultural implications of piracy shaped many of today’s understanding of pirates and their impact on the triangular trade economically. However, historians that study piracy today often study the understanding of pirates in regard to social and cultural movements. In the mid-to-late- 1900s new cultural waves of understanding clouded academia and led to some of the worlds most influential writers of pirate history. Historians of the social turn inspired by new movements for civil rights and women’s rights, formulate new lenses to view events in the past.
+
+The Social turn of the 1960s transformed pirate history into a bottom-up narrative. Meaning that historians started to write about pirates in regard to the lowliest of cabin boys rather than that of captains. The idea of studying the life of a commoner stood well with much of society at the time. Bottom-up history also includes the study of once marginalized groups of people, such as Africans and Women. 
 
-You have freedom to roam from the narrow topic of your project to explore how different authors/schools of thought have impacted scholarly approaches over time. However, make sure that in the end you clearly relate how this discussion relates to the subject of your project and/or your choices in methods or interpreative models.
+Fallowing the social turn in the 1960s, the cultural turn further transformed history into a more inclusive discipline in regard to poorer socio-economic classes. Marcus Rediker, a specialist in the history of pirates, takes a more Marxist approach to this history. In his article, “Under the Banner King Death”: The Social world of Anglo-American Pirates, 1712 to 1726”, Rediker explores the “cultural dimensions of piracy” through the experiences and the social hierarchy of pirates. Rediker is considered a Marxist historian because he offers an approach that favors that of the working class rising up and overthrowing the bourgeoisie. This approach mirrors the popular notion that pirates were ruthless and unlawful beings, which fit in no society except that on the sea. Unlike other historians, Rediker examines the social order of piracy, and determine the type of hierarchy a pirate ship may undergo through a socialist lens.
```

Comment:
Watch the spelling, 'Fallowing'. Definitely want to iron those out if you want to use it for a resume/cv/portfolio